.meshrc.yaml

```diff
@@ -128,6 +135,15 @@ sources:
             responseTypeName: UpdateSampleNotes
             headers:
               Cookie: "{context.headers['cookie']}"
+          - type: Mutation
+            field: KickoffWorkflow
+            path: /samples/{args.sampleId}/kickoff_workflow
+            method: POST
+            requestSample: {"inputs_json[start_from_mngs]": "true", "workflow": "amr", "authenticityToken": "token"}
```

Comment:
I believe we also need to federate kicking off a cg workflow from an existing mngs sample

Reply:
Removed that bit code from this PR. Will address it in https://github.com/chanzuckerberg/czid-graphql-federation-server/pull/9